Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.Tests/Lib/Helpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
```

Comment:
You're not using this namespace, so it can be removed.

Reply:
Removed